### PR TITLE
#159 Handle "No schedule" notifications better when car gets disconnected

### PR DESCRIPTION
**Pull request by ArdJonker**

### CODE REVIEW

File: v2g_liberty.py [hunk 1]
```diff
@@ -276,12 +283,40 @@ def clear_notification(self, identification: dict):
         except:
             self.log(f"Could not clear notification: exception on {recipient}.")
 
+    def reset_no_new_schedule(self):
+        """ Sets all errors to False and removes notification / UI messages
+
+        To be used when the car gets disconnected, so that while it stays in this state there is no
+        unneeded "alarming" message/notification.
+        Also, when the cr returns with an SoC below the minimum no new schedule is retrieved and
```
Comment (Flix6x) [inline v2g_liberty.py:291]: The *a* switched to another car.

**Review by Flix6x — APPROVED** [review]
Difficult for me to verify the logic, but it looks sensible.

File: v2g_liberty.py [hunk 2]
```diff
@@ -291,17 +326,30 @@ def handle_no_new_schedule(self, error_name: str, error_state: bool):
                     Remove from UI immediately
                     If notification has been sent:
                         Notify user the situation has been restored.
+
+            Parameters
+            ----------
+            reset : bool, optional
+                    reset is ment for the situation where the caar gets disconnected and all
```
[the remark this answers is not in the document]
Comment (ArdJonker) [reply v2g_liberty.py:333]: Ohh, shit. I said i was going to be more precise on this. Sorry!

File: v2g_liberty.py (hunk 1 above)
Comment (ArdJonker) [reply v2g_liberty.py:291]: :-)

**Pull request merged**